### DISCUSSION TITLE
Does it support k8s node failure chaos?

### DISCUSSION THREAD

**Discussion by AllenZMC** [chaos-mesh/chaos-mesh#3693] [Q&A]
Does it support k8s node failure chaos?

**Comment by STRRL** [comment]
Chaos Mesh does NOT support the generally "k8s node failure" yet, because it's not specific, and the "node failure" may be a complex situation.

But you could still use `chaosd` and `PhysicalMachineChaos` to inject different chaos in the node level, try to attach the k8s node.

**Reply by STRRL** [reply]
see: https://chaos-mesh.org/docs/chaosd-overview/

**Reply by AllenZMC** [reply]
I see that chaosblade supports process chaos, does chaosmesh support？
see https://chaosblade.io/en/docs/experiment-types/k8s/blade%20create%20k8s%20node-process
https://chaosblade.io/en/docs/experiment-types/k8s/blade%20create%20k8s%20container-process
